addons/mail/static/tests/new/chat_window/chat_window_manager_tests.js

```diff
@@ -132,6 +132,9 @@ QUnit.test(
         assert.containsOnce(target, ".o-mail-chat-window:eq(0):contains(channel-A)");
         assert.containsOnce(target, ".o-mail-chat-window:eq(1):contains(channel-D)");
         assert.containsOnce(target, ".o-mail-chat-window-hidden-button:contains(2)");
+        await waitUntil(
```

Comment:
It's strange to have `waitUntil` not directly after use action, as if all the previous assertion can be done during inconsistent state?
Wouldn't `.o-mail-chat-window:eq(1):contains(channel-D)` above potentially fail too if waited not long enough?